content/docs/0.9.x/operate/advanced_install_options/index.md

```diff
@@ -6,10 +6,26 @@ weight: 20
 
 ## Advanced Install Options: Install Keptn using the Helm chart
 
-When executing `keptn install`, Keptn is installed via a Helm chart, which can also be used directly.
-For this, the [Helm CLI](https://helm.sh) is required.
+When executing `keptn install`, Keptn is installed via a Helm chart, which can also be done bypassing the `keptn CLI`.
```

Comment:
for me, the sentence "which can also be done bypassing the keptn CLI" is not clear.
suggestion: Instead of using the Keptn CLI for installation, you can also use the Helm CLI directly.

Reply:
sry, ... totally agree that the sentence is wired. changed it.